contracts/burner/src/contract.rs

```diff
@@ -1,12 +1,12 @@
 use cosmwasm_std::{
-    attr, Api, BankMsg, Binary, Deps, Env, HandleResponse, InitResponse, MessageInfo,
-    MigrateResponse, Order, Querier, StdError, StdResult, Storage,
+    attr, BankMsg, Binary, Deps, DepsMutMut, Env, HandleResponse, InitResponse, MessageInfo,
```

Comment:
```suggestion
    attr, BankMsg, Binary, Deps, DepsMut, Env, HandleResponse, InitResponse, MessageInfo,
```

Reply:
Yup, fixed this. And also rebuilt the test contract.

Getting ready to merge.